gcloud/pubsub/test_client.py

```diff
@@ -44,7 +81,7 @@ def test_list_topics_no_paging(self):
         self.assertEqual(next_page_token, None)
 
         self.assertEqual(len(conn._requested), 0)
```

Comment:
OK. Care to remove `conn = client.connection = _Connection()` and it's cousins throughout (where they are no longer needed)?